terracumber_config/tf_files/SUSEManager-4.1-NUE.tf

```diff
@@ -0,0 +1,183 @@
+// Mandatory variables for terracumber
+variable "URL_PREFIX" {
+  type = "string"
+  default = "https://ci.suse.de/view/Manager/view/Manager-4.1/job/manager-4.1-cucumber-NUE"
+}
+
+// Not really used as this is for --runall parameter, and we run cucumber step by step
+variable "CUCUMBER_COMMAND" {
+  type = "string"
+  default = "export PRODUCT='SUSE-Manager' && run-testsuite"
+}
+
+variable "CUCUMBER_GITREPO" {
+  type = "string"
+  default = "https://github.com/SUSE/spacewalk.git"
+}
+
+variable "CUCUMBER_BRANCH" {
+  type = "string"
+  default = "Manager-4.1"
+}
+
+variable "CUCUMBER_RESULTS" {
+  type = "string"
+  default = "/root/spacewalk/testsuite"
+}
+
+variable "MAIL_SUBJECT" {
+  type = "string"
+  default = "Results 4.1-NUE $status: $tests scenarios ($failures failed, $errors errors, $skipped skipped, $passed passed)"
+}
+
+variable "MAIL_TEMPLATE" {
+  type = "string"
+  default = "../mail_templates/mail-template-jenkins.txt"
+}
+
+variable "MAIL_SUBJECT_ENV_FAIL" {
+  type = "string"
+  default = "Results 4.1-NUE: Environment setup failed"
+}
+
+variable "MAIL_TEMPLATE_ENV_FAIL" {
+  type = "string"
+  default = "../mail_templates/mail-template-jenkins-env-fail.txt"
+}
+
+variable "MAIL_FROM" {
+  type = "string"
+  default = "galaxy-ci@suse.de"
+}
+
+variable "MAIL_TO" {
+  type = "string"
+  default = "galaxy-ci@suse.de"
+}
+
+// sumaform specific variables
+variable "SCC_USER" {
+  type = "string"
+}
+
+variable "SCC_PASSWORD" {
+  type = "string"
+}
+
+variable "GIT_USER" {
+  type = "string"
+  default = null // Not needed for master, as it is public
+}
+
+variable "GIT_PASSWORD" {
+  type = "string"
+  default = null // Not needed for master, as it is public
+}
+
+provider "libvirt" {
+  uri = "qemu+tcp://ramrod.mgr.suse.de/system"
+}
+
+module "cucumber_testsuite" {
+  source = "./modules/cucumber_testsuite"
+
+  product_version = "4.1-nightly"
+  
+  // Cucumber repository configuration for the controller
+  git_username = var.GIT_USER
+  git_password = var.GIT_PASSWORD
+  git_repo     = var.CUCUMBER_GITREPO
+  branch       = var.CUCUMBER_BRANCH
+
+  cc_username = var.SCC_USER
+  cc_password = var.SCC_PASSWORD
+
+  images = ["centos7", "opensuse150", "sles15sp1", "sles15sp2", "ubuntu1804"]
+
+  use_avahi    = false
+  name_prefix  = "suma-41-"
+  domain       = "mgr.suse.de"
+  from_email   = "root@suse.de"
+
+
+  portus_uri = "portus.mgr.suse.de:5000/cucutest"
+  portus_username = "cucutest"
+  portus_password = "cucusecret"
+
+  server_http_proxy = "galaxy-proxy.mgr.suse.de:3128"
+
+  host_settings = {
+    ctl = {
+      provider_settings = {
+        mac = "AA:B2:93:00:00:80"
+      }
+    }
+    srv = {
+      provider_settings = {
+        mac = "AA:B2:93:00:00:75"
+      }
+    }
+    pxy = {
+      provider_settings = {
+        mac = "AA:B2:93:00:00:81"
+      }
+    }
+    cli-sles12sp4 = {
```

Comment:
We have sles12sp4 at Head and 4.0 as well, so I just did the same.

But why do we have it this way? Is it a sumaform limitation?

Reply:
It's not sles12sp4 at all. It's a badly chosen name..

@rjmateus could we have this fixed once for good, and rename it e.g. `client` ?

Reply:
@Bischoff we have a card to refactor it: https://github.com/SUSE/spacewalk/issues/10730
I can change just the names and them refactor the cucumber testsuite module. But now is not the time to do breaking changes, since we are in process of GA.